tests/e2e/UsingGKE.md

```diff
@@ -0,0 +1,160 @@
+Then in the Roles drop-down list for that account, find the Kubernetes Engine group and select the role Kubernetes
+Engine Admin. The Roles listing for your account will change to Multiple.
+
+## Step 2: Get cluster credentials
+```
+gcloud container clusters get-credentials ${CLUSTER_NAME} --zone ${ZONE} --project ${PROJECT_NAME}
+```
+
+## Step 3: Create Clusterrolebinding
+```
+kubectl create clusterrolebinding myname-cluster-admin-binding    --clusterrole=cluster-admin    --user="<user_email>"
+```
+* user_email should be the one you use to log in gcloud command. You can do `gcloud info` to find out current user info.
+
+
+## Step 4: Export test script variables
+
+**Option 1:** Build your own images.
+
+```
+# Customize .istiorc.mk (at the top of the istio.io/istio source tree) with your HUB and optional TAG
+# it allows you to customize Makefile rules. For example:
+cat .istiorc.mk
+HUB=costinm
+TAG=mybranch
+
+# Build images on the local docker.
+make docker
+
+# Push images to docker registry
+make push
+
+# the hub/tag set in your .istiorc.mk will be used by the test.
+
+```
+
+**Option 2:** Already committed changes to istio/istio master branch
+NOTE: SHA used as TAG is one that is already committed on istio/istio. You can pick any SHA you want.
+```
+export HUB="gcr.io/istio-testing"
+export TAG="d0142e1afe41c18917018e2fa85ab37254f7e0ca"
+```
+
+**Option 3:** Testing local changes
+
+If you want to test on uncommitted changes to master istio:
+* Create a PR with your change.
+* This will trigger istio-presubmit.sh. At the end of this script, it creates docker images for mixer, pilot, ca, with
+your changes and upload them to container registry. See the logs of this istio-presubmit.sh and at the end there must
+be a SHA which you need to copy and set it as a GIT_SHA. Example from a log: the SHA to copy is marked in **bold**
+
+	I1207 04:42:40.881] **0077bb73e0b9d2841f8c299f15305193e42dae0d**: digest: sha256:6f72528d475be56e8392bc3b833b94a815a1fbab8a70cd058b92982e61364021 size: 528
+
+* Then set the export variables again
+```
+export HUB="gcr.io/istio-testing"
+export TAG="<sha copied from the logs of istio-presubmit.sh>"
+```
+
+## Step 5: Run
+
+From the repo checkout root directory
+
+```bash
+make e2e_all E2E_ARGS="--skip_cleanup"
+```
+
+Each test has its own directory and can be run independently as a
+go_test target. The script has a number of options:
+
+* `--skip_cleanup` - to skip cleanup steps
+* `--namespace <namespace>` : If you don't specify `namespace`, a random namespace is generated for each test.
+* `--verbose <debug level noise from proxies>`
+* `--istioctl <local istioctl path>`: Use local istioctl binary (i.e. `${GOPATH}/out/linux_amd64/release/istioctl`).
+* `--istioctl_url <remote istioctl url>`: If local path is not defined, download istioctl from a remote location.
+* `--use_local_cluster`: If running on minikube, this should be set to true.
```

Comment:
should we also add vagrant here?

Reply:
We already have a different page for vagrant so I don't think it is needed here.